packages/block-library/src/image/image.js

```diff
@@ -636,7 +636,7 @@ export default function Image( {
 								} }
 							>
 								{ _x(
-									'Alt',
+									'Alternative text',
 									'Alternative text for an image. Block toolbar label, a low character count is preferred.'
```

Comment:
Should the translation text be revised? Especially the low character count bit now that it's longer.